tests/cli/dorq/test_entrypoint.py

```diff
@@ -276,3 +276,27 @@ def test_with_flag(self, entrypoint, mock_login_action, flag, expected_runtime):
         mock_login_action.assert_called_with(
             config=ANY, url=ANY, token=ANY, runtime_name=expected_runtime
         )
+
+
+class TestVersion:
+    @staticmethod
+    def test_shows_version(capsys, entrypoint, monkeypatch):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏This test checks for the version help text
```suggestion
    def test_shows_version_help_text(capsys, entrypoint, monkeypatch):
```

Reply:
[f78052b](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/252/commits/f78052bab0085849c0cac7fdd981ac8d743c423f)